hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/components/HawkbitUIErrorHandler.java

```diff
@@ -38,10 +42,21 @@ public void error(final ErrorEvent event) {
 
         final Optional<Page> originError = getPageOriginError(event);
 
+        final HawkbitErrorNotificationMessage message = buildNotification(getRootExceptionFrom(event));
```

Comment:
`buildNotification(...)` only adds a log statement can't you move the log-statement some where else and remove the seperate method?